assets/overlays/openstack-manila/generated/hypershift/controller.yaml

```diff
@@ -346,6 +326,27 @@ spec:
         volumeMounts:
         - mountPath: /csi
           name: socket-dir
+      initContainers:
```

Comment:
this was tested and this can't work as is because init containers need to finish (successfully) for other containers to start. This one doesn't finish as it leaves an open socket.